app/config.c

```diff
@@ -214,6 +215,8 @@ static void parse_option(const char *option, const char *params)
         } else if (strncmp(params, "rr", 3) == 0 || strncmp(params, "one", 4) == 0) {
             cpu_mode = ONE;
         }
+    } else if (strncmp(option, "dark", 5) == 0) {
```

Comment:
In #494, 01e3 had a point, even if for now, the user only requested a dark mode as you mentioned: I too would prefer future-proofing the command-line option, by checking for e.g. "colors=dark" or "theme=dark".

Reply:
There is no request for this and I really don't want to implement many color schemes. Let's keep it simple.